packages/Liquid-Tests.package/ObjectRepoTests.class/instance/testErrorHandling.st

```diff
@@ -0,0 +1,14 @@
+tests
+testErrorHandling
+	| server port url client dummy |
+	port := 10000 random.
```

Comment:
[dust] I'm not quite sure whether choosing a random port is ok in a unit test. What would happen if the port is already used by another application?

Reply:
Squeak's WebServer has an interesting behaviour where, when it attempts to listen on an already-taken port, it doesn't throw an error but just silently dies. Requests will then go to the old server, which is probably a leftover from an old test that wasn't cleaned up properly.
This problem doesn't occur with random ports, and (depending on the range of ports we draw from) the chance of conflicts is pretty slim here, I agree that this isn't the most lovable solution.

Another ideas for solving this would be:
1. Use fixed ports and implement proper clean-up.
2. Use some global auto-incrementing port number

Reply:
Continued in #104